traffic_ops/traffic_ops_golang/config.go

```diff
@@ -62,6 +62,8 @@ type ConfigTrafficOpsGolang struct {
 	LogLocationEvent       string `json:"log_location_event"`
 	Insecure               bool   `json:"insecure"`
 	MaxDBConnections       int    `json:"max_db_connections"`
+	MojoliciousBacklogSize int    `json:"mojolicious_backlog_size"`
```

Comment:
I'm still not sold that this should be called 'MojoliciousBacklogSize' instead of just 'MaxBacklogSize'